cleanup/cleanup.sh

```diff
@@ -1,3 +1,95 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Might use `#!/usr/bin/env bash`. The plugin doesn't work with `bash` version < 4.